src/mongoose-cleaner.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
```

Comment:
I skipped the omit, seemed unnecessary to clone it twice just to remove a property.